test/functional/page_objects/visualize_page.js

```diff
@@ -248,17 +248,28 @@ export function VisualizePageProvider({ getService, getPageObjects }) {
 
     async setComboBox(comboBoxSelector, value) {
       const comboBox = await testSubjects.find(comboBoxSelector);
-      await this.setComboBoxElement(comboBox, value);
+      await this.setComboBoxValue(comboBox, value);
+      await this.closeComboBoxOptionsList(comboBoxSelector);
       await PageObjects.common.sleep(1000);
     }
 
-    async setComboBoxElement(element, value) {
-      const input = await element.findByTagName('input');
+    async setComboBoxElement(comboBox, value) {
+      await this.setComboBoxValue(comboBox, value);
+
+      // Close list.
+      const isOptionsListOpen = await testSubjects.exists('comboBoxOptionsList');
```

Comment:
Can `closeComboBoxOptionsList` function be used here instead?

Reply:
I don't think so, since that method requires you to pass in a data-test-subj selector, and we don't have access to one in this method (we only have access to the combo box element itself).